### PR TITLE
fix: line chart RTL mode color fillbar 

### PR DESCRIPTION
**Pull request by harsh9975**
image

## Previous Behavior
Color fillbars in line chart in RTL overlap with y axis

## New Behavior
Color fill bar is rendering correctly.

## Related Issue(s)
#29862

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/d82568581b73996acbfdf78966e55572e0f869f6'>d82568581b73996acbfdf78966e55572e0f869f6</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-charting-f9607dcd-7f93-4138-9042-5211cee63493.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-charting-f9607dcd-7f93-4138-9042-5211cee63493.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29893&commitId=623e5fd32d43587fcfb192bedb5cfa05b4887b6d&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Multiple - RTL.default.chromium.png|6662| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (AtishayMsft) [reply change/@fluentui-react-charting-f9607dcd-7f93-4138-9042-5211cee63493.json:1]: The issue seen in the baseline is fixed with this PR.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**